src/frontend/src/pages/search/form/Within.vue

```diff
@@ -35,11 +36,15 @@ export default Vue.extend({
 	computed: {
 		withinOptions(): Option[] {
 			const {enabled, elements} = UIStore.getState().search.shared.within;
-			return enabled ? elements.filter(corpusCustomizations.search.within.include) : [];
+			return enabled ? elements.filter(element => corpusCustomizations.search.within.includeSpan(element.value)) : [];
 		},
 		within: {
-			get(): string|null { return PatternStore.getState().extended.within; },
-			set: PatternStore.actions.extended.within,
+			get(): string|null {
+				return PatternStore.getState().shared.within;
+			},
+			set(v: string|null) {
+				Vue.set(PatternStore.getState().shared, 'within', v);
```

Comment:
Waarom niet een setter? Zo krijg je volgens mij steeds warnings in de console

Reply:
Luiheid :) Fixed!